src/pages/tonieboxes/TonieboxesPage.tsx

```diff
@@ -45,11 +45,11 @@ export const TonieboxesPage = () => {
         <StyledContent>
           <h1>{t('tonieboxes.title')}</h1>
           <p><Alert
-                          message="Warning! PoC state."
-                          description= "This page is still in development and is in state of a PoC! No data is saved. Box appearance is hardcoded and if you change it, after reload of page it's reset."
-                          type="warning"
-                          showIcon
-                    />
+            message="Warning! PoC state."
```

Comment:
the message from the alert should be translated

Reply:
its removed, was just for the time, when the PoC was not yet completed. Now it's "completed" and in normal develop state